PWGCF/FEMTOSCOPY/FemtoDream/AliAnalysisTaskFemtoDreamPhi.h

```diff
@@ -64,7 +64,7 @@ class AliAnalysisTaskFemtoDreamPhi : public AliAnalysisTaskSE {
   AliFemtoDreamPartCollection *fPartColl;  //!
   AliAODTrack **fGTI;                      //!
   int fTrackBufferSize;                    //
-  ClassDef(AliAnalysisTaskFemtoDreamPhi, 3)
+  ClassDef(AliAnalysisTaskFemtoDreamPhi,4)
```

Comment:
since you didn't add a member that needs to be streamed you can leave the counter as it was